File: Grupo 2-3/C Sharp/Ejercicios/dolares.cs
using System;

class Principal
{
    static int Conversion(int Dolares, int TipoCambio)
    {
        int Pesos;
        Pesos = Dolares * TipoCambio;
        return Pesos;
    }

    static void Main( )
    {

        int Cantidad = Convert.ToInt32(Console.ReadLine());

        int Precio = Convert.ToInt32(Console.ReadLine());

        int Equivalencia = Conversion(Cantidad, Precio);

        Console.WriteLine("{0}", Equivalencia);


    }
}

File: Grupo 2-1/Lenguaje C/funciones/ Ejercicios/dolares.c
#include <stdio.h>

int conversion(int dolares, int tipoCambio){
    int pesos;
    pesos = dolares * tipoCambio;
    return pesos;
}
int main()
{
    int cantidadDolares, precioDolar;
    int equivalenciaPesos;
    scanf ("%d", &cantidadDolares);
    scanf ("%d", &precioDolar);
    equivalenciaPesos = conversion(cantidadDolares, precioDolar);  
    printf("%d", equivalenciaPesos);
    return 0;
}
